src/components/Calendar/hooks/useHeaderArrowNav.js

```diff
@@ -0,0 +1,61 @@
+import { useState, useMemo, useEffect, useCallback } from 'react';
+import { RIGHT_KEY, LEFT_KEY } from '../../../libs/constants';
+
+export default function useHeaderArrowNav({
+    disableNextMonth,
+    disablePreviousMonth,
+    refs,
+    firstIndex,
+    lastIndex,
+    delta,
+}) {
+    const [enableNavKeys, setEnableNavKeys] = useState(false);
```

Comment:
I think you can do without this state because it is the same as (focusedItemIndex === -1). In the clearFocusedItems () you can do setFocusedItemIndex (-1)

Reply:
no.. focusedIndex cannot be used to control enabled states...